mpl_interactions/generic.py

```diff
@@ -480,3 +483,103 @@ def _onselect(self, verts):
 
     def _ipython_display_(self):
         display(self.fig.canvas)
+
+
+def hyperslicer(
+    arr,
+    cmap=None,
+    norm=None,
+    aspect=None,
+    interpolation=None,
+    alpha=None,
+    vmin=None,
+    vmax=None,
+    origin=None,
+    extent=None,
+    autoscale_cmap=True,
+    filternorm=True,
+    filterrad=4.0,
+    resample=None,
+    url=None,
+    ax=None,
+    slider_format_string=None,
+    title=None,
+    figsize=None,
+    display=True,
+    force_ipywidgets=False,
+    **kwargs,
+):
```

Comment:
obvi we will need a docstring one day. Can probs straight copy most of it from `interactive_imshow`